src/display/process/allProcs.go

```diff
@@ -130,6 +133,11 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 		w, h := ui.TerminalDimensions()
 		myPage.Grid.SetRect(0, 0, w, h)
 		help.Resize(w, h)
+		if sendSignal {
+			signals.SetRect(0, 0, w/6, h)
+			myPage.Grid.SetRect(w/6, 0, w, h)
+			ui.Render(signals)
+		}
```

Comment:
I made it explicit because the rest of the UI also needs to be rendered after this. I could make it an else-if branch, but would need to repeat the call to render the grid. Or I could have a nested if under the existing else branch to check if the signal menu is open. I felt this was a better alternative. Let me know what you feel is best

Reply:
The issue is, we'd want the help page to take precedence, even if pressed when kill is selected. As of now that hinders the UI as both of them get rendered.

Besting under the existing else branch is a good idea though. Go for it!

Reply:
Okay this is fixed now. I also had to change when updateUI is called for the help window keybinding, because for some reason handling the same event twice wasn't working well with the signal selector (exiting vie <Esc> was fine, but exiting via "?" was not)